server/api/controller/cron.js

```diff
@@ -145,6 +146,86 @@ const sendPlanningAlerts = () => {
 		});
 };
 
+const planToEmail = (plan, map) => {
+	return {
+		id: plan.get('id'),
+		title: plan.get('plan_display_name'),
+		city: plan.get('PLAN_COUNTY_NAME'),
+		text: plan.get('goals_from_mavat'),
+		status: plan.get('status'),
+		areaChange: plan.describeHousingChange(),
+		map: 'data:image/gif;base64,'+ map,
+		link:`${this.baseUrl}plan/${plan.get('id')}`
+	};
+}; 
+
+const alertToEmail = (alert) => {
+	const alertTitle = `תוכניות חדשות בקרבת ${alert.get('address') ||  'תחומי הענין שלך'}`;
+	return {
+		alert:{
+			title: alertTitle,
+			unsubscribeLink: `${this.baseUrl}alerts/unsubscribe/${alert.unsubsribeToken()}`
+		}
+	};
+};
+
+const sendDigestPlanningAlerts = async () => {
+	// Send emails for each user, by new plans in his area, that
+	// have been added since he last received a digest email
+	// sendPlanningAlerts(req, res, next) {id
+	Log.info('Running digest send planning alert');
+	const lastSentDifference = 150;
+	const maxAlertsToSend = 5;
+	const timeDifference = moment.duration(lastSentDifference, 'd');
+	const date = moment().subtract(timeDifference);
+
+	try {
+		const alertToNotify =  await Alert.getAlertToNotify(date, 1);
+		const alertGeom = alertToNotify.get('geom');
+		const alertPlans = await Plan.getPlansByGeometryThatWereUpdatedSince(alertGeom, date);
+		Log.debug(`Got ${alertPlans[0].length} for alert ${alertToNotify.id}`);
+		const maps = await Bluebird.mapSeries(alertPlans, plan => {
+			const planGeom = plan.get('geom');
+			if(!planGeom) return plan;
+			const centroid = Turf.centroid(planGeom);	
+			return drawStaticMapWithPolygon(
+				centroid.geometry.coordinates[1],
+				centroid.geometry.coordinates[0],
+				planGeom
+			);
+		});
+
+		const emailAlertParams = alertToEmail(alertToNotify);
+
+		const emailPlanParams = {
```

Comment:
Seems like a weird thing for me, it's transfering an array to a map with indices as keys? Like, it's still an array...
I think it's best keeping the array and iterating on it later rather than creating a weird array-like map.
Other than it being better SE concept in my opinion, it allows us to change the number of items that we present in a single digest with ease.